src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import {useParams} from "react-router-dom";
+import axios from "axios";
+
+function Subreddit() {
+    const { id } = useParams();
+
+    const [reddits, setReddits] = useState({});
+
+    useEffect(() => {
+        async function getRedditData() {
+            try {
+                /*const result = await axios.get(`https://www.reddit.com/r/memes/about.json`)*/
+                const result = await axios.get(`https://www.reddit.com/r/${id}/about.json`)
+                console.log(result.data.data)
+                setReddits(result.data.data)
+
+            } catch (e) {
+                console.error(e)
+            }
+        }
+
+        getRedditData()
+    }, []);
+
+    return (
+
+        <div className="outer-container">
+            <div className="inner-container">
+                {Object.keys(reddits).length > 0 && (
```

Comment:
De check kan dan na de vorige comment zo worden:
```suggestion
                {reddits && (
```